arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender.dts

```diff
@@ -89,6 +89,18 @@
 	};
 };
 
+&blsp_i2c1 {
+	status = "okay";
+
+	touchscreen: novatek@62 {
+		compatible = "novatek,nt36525";
```

Comment:
Is it 36525 really..?

Reply:
I'm actually not sure, but defining this actually works.

Reply:
I bet downstream nvt-ts driver prints something, I remember it vomiting a lot of logs to dmesg